rt/mlc/TESTDC1.MLC

```diff
@@ -0,0 +1,461 @@
+* 07/19/06 RPI 368 - support S scale factor for F, H, EH, DH, and LH      
+* 08/03/06 RPI 388 - test fix dup count for E, D, F and FD                     
+* 09/21/06 RPI 458 - test DC S(rel and off(base)) constants                            
+* 12/09/06 RPI 407 - test DC DD, ED, and LD (min) and (max) constants                  
+* 01/19/07 RPI 538 - test duplication factors for P and all types                           
+* 03/01/07 RPI 555 - test LQ quad word hex                                                  
+* 03/16/07 RPI 578 - test corrected M3 complex mult S type
+* 04/01/07 RPI 567 - add CCW, CCW0, and CCW1 tests                
+* 01/10/08 RPI 777 - add decimal and scale for P/Z, add sign for Z 
+* 04/22/08 RPI 833 ADD STRING QUOTES FOR HLASM COMPATIBILITY 
+* 04/24/08 RPI 840 TEST SPACES IGNORED IN P AND Z DATA FIELDS 
+* 08/09/08 RPI 893 TEST SY LONG DISP, F'Unn', H'Unn'                          
+* 11/10/08 RPI 946 CHANGE DS TO DC FOR INIT TO ZEROS                          
+* 06/30/09 RPI 1044 CHANGE VDX DC VD(VDX) TO USE TESTSUB1                     
+* 01/09/10 RPI 1101 AFHY OVERFLOW WITH NO ERRORS
+* 01/25/11 RPI 1139 C2X ARG IN QUOTES FOR NOALLOW
+* 2022-05-03 update copyright, add to z390\tests, fix SETC '&&' to '&'
+*********************************************************************
+         PRINT DATA
+TESTDC1  RT1   MAIN
+M1       DC    AL1(1),B'10',C'3',FL1'4',HL1'5',P'6',X'7',Z'8',X'FF'
+V1E      DC    X'0102F304056C07C8FF' RPI 777
+V1A      DC    X'01023304056C07C8FF' RPI 777
+M2       DC    L'1',D'1',E'1',X'FF'
+V2       DC    X'41100000000000003300000000000000'
+         DC    X'4110000000000000'
+         DC    X'41100000',X'FF'
+M3       DC    S(1),S(2),S(M3),S(1(2)),X'FF'
+V3       DC    X'00010002',AL2(X'D000'+(M3-TESTDC1-8)),X'2001',X'FF'
+M4       DC    V(TESTDC1),2V(TESTDC1),X'FF'
+V4       DC    3A(TESTDC1),X'FF'
+         DS    0F
+V5       DC    X'AA',0H,X'BBCC' FORCE 1 ALIGNMENT BYTE WITHIN DC
+L1       DC    L'1'
+         DC    XL8'00'
+L2       DC    L'2'
+TE       DC    2E'0'
+TD       DC    2D'0'
+TL       DC    2L'0'
+TFD      DC    2FD'0'
+TF       DC    2F'0'
+TFE      EQU   *
+         DS    0D              RPI 1073 REQUIRED IF USING NOALIGN
+TES      DC    0E,2F'0'    2E  RPI 946         
+TDS      DC    0D,2FL8'0'  2D  RPI 946         
+TLS      DC    0L,2FL16'0' 2L  RPI 946         
+TFDS     DC    0FD,2FL8'0' 2FD RPI 946         
+TFS      DC    0F,2F'0'    2F  RPI 946         
+TFSE     EQU   *    
+CCW      CCW   OP,DATA,FLAGS,LEN
+CCWV     DC    AL1(OP),AL3(DATA),AL1(FLAGS,0),AL2(LEN)
+CCW0     CCW0  OP,DATA,FLAGS,LEN
+CCW0V    DC    AL1(OP),AL3(DATA),AL1(FLAGS,0),AL2(LEN)
+CCW1     CCW1  OP,DATA,FLAGS,LEN
+CCW1V    DC    AL1(OP),AL1(FLAGS),AL2(LEN),A(DATA)
+DATA     DC    D'0'
+OP       EQU   X'12'
+FLAGS    EQU   X'34'
+LEN      EQU   X'5678'
+         RT1   START
+* verify ds/dc alignment between fields
+         RT1   CCE,V5,=X'AA00BBCC'
```

Comment:
Question. Because of the ASSIST additions, won't the filler byte generated by the "0H" in the V5 declaration be X'F6', not X'00'? If so, recommend making sure the TESTDC1 test uses the NOINIT option.